includes/admin/shortcodes/shortcode-give-donor-wall.php

```diff
@@ -49,19 +49,18 @@ public function define_fields() {
 			array(
 				'type'        => 'listbox',
 				'name'        => 'order',
-				'label'       => esc_attr__( 'Columns:', 'give' ),
+				'label'       => esc_attr__( 'Order:', 'give' ),
 				'tooltip'     => esc_attr__( 'Sets the number of forms per row.', 'give' ),
```

Comment:
@ravinderk I overlooked one tooltips description which is incorrect. Please update the tooltip for `Order` so that it reads: `Sets the order in which donors appear.`